docs/schemas/profile.json

```
@@ -0,0 +1,140 @@
+          },
+          "additionalProperties": false
+        }
+      ],
+      "description": "Xdebug execution trace data with specification URL. Provides detailed function-by-function execution flow that standard PHP profiling cannot capture, essential for identifying complex performance issues, recursive calls, and execution path bottlenecks."
+    },
+    "php": {
+      "type": "object",
+      "properties": {
+        "backtrace": {
+          "type": "array",
+          "description": "PHP backtrace information captured using debug_backtrace() function",
```

Biome (2.1.2)

[error] The key description was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)